com.bktus.gpgfrontend.yaml

```diff
@@ -0,0 +1,62 @@
+app-id: com.bktus.gpgfrontend
+runtime: org.kde.Platform
+runtime-version: "6.5"
+sdk: org.kde.Sdk
+command: GpgFrontend
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=home
+  - --filesystem=/run/user
```

Comment:
Is this for gpg agent socket? Then what you want is `--socket=gpg-agent` which gives access to `gpgconf --list-dirs agent-socket`